src/views/tunnel_setup.mustache

```diff
@@ -23,8 +23,7 @@
       <div id="error-setup" class="error hidden">
         An error occured when setting up the subdomain
       </div><br>
-      <!-- https://github.com/mozilla-iot/gateway/issues/159 -->
-      <!--<button id="skip-subdomain-button" type="button">Skip</button>-->
+      <button id="skip-subdomain-button" type="button">Skip</button>
```

Comment:
On the mockups this is a small link underneath the next button, rather than a button, to try to encourage people to choose a subdomain rather than skip it. This is because 1) It's more secure than plain HTTP 2) It will allow remote access through a firewall and 3) it will allow us to count the gateway in our analytics.

I suggest making it a small link underneath the button. An alternative might be to add "(recommended)" after "Choose a secure web address for your gateway", but I suspect the former will be more effective.